tests/system_tests/test_core/test_offline_sync_beta.py

```diff
@@ -197,6 +197,27 @@ def test_syncs_run(tmp_path, wandb_backend_spy, runner: CliRunner):
         assert "test_file.txt" in files
 
 
+def test_syncs_resumed_run(wandb_backend_spy, runner: CliRunner):
```

Comment:
TODO: This test is unreliable. When I run 14 instances of it in parallel, sometimes the FileStream uploads for the resumed runs start at the wrong offset, overwriting past data. It seems the backend can return an old offset when resuming runs in rapid succession.

Reply:
I tried again and couldn't reproduce the problem, so perhaps this was a backend issue that got fixed.